pkg/metrics/statsd/statsd.go

```diff
@@ -91,7 +83,7 @@ func (s *statsd) emitMetric(m *metric) {
 		m.dims = map[string]string{}
 	}
 	m.dims["location"] = s.env.Location()
-	m.dims["hostname"] = s.hostname
+	m.dims["hostname"] = s.env.Hostname()
```

Comment:
just trying to understand reasoning why this moved into `env`? since in both cases in env it is set to `os.Hostname()` what do we gain by this change?

Reply:
This change is made so that there is only one place where we call `os.Hostname()`. I think it makes sense for `InstanceMetadata` to know about the hostname. In this case, we used it in the [payload hydration method](https://github.com/Azure/ARO-RP/blob/b48b157e216fcadff988b2362ec3674d6df25a57/pkg/util/log/audit/audit.go#L135). 

See this [feedback](https://github.com/Azure/ARO-RP/pull/1224#pullrequestreview-551755888).